mmdeploy/utils/device.py

```diff
@@ -11,6 +11,7 @@ def parse_device_id(device: str) -> int:
 
     Returns:
         int: The parsed device id, defaults to `0`.
+            If device is 'cpu' it will return `-1`.
     """
     if device == 'cpu':
```

Comment:
The docstring may get updated. `parse_device_id` parse both cpu and gpu devices

Reply:
Updated the documentation and implementation a bit.